contentgrid-spring-data-rest/build.gradle

```diff
@@ -42,7 +42,10 @@ dependencies {
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.boot:spring-boot-starter-web'
     testImplementation 'jakarta.transaction:jakarta.transaction-api'
+    testImplementation 'org.springframework.security:spring-security-test'
+    testImplementation 'org.springframework.boot:spring-boot-starter-oauth2-resource-server'
     testImplementation project(':contentgrid-spring-boot-autoconfigure')
+    testImplementation project(':contentgrid-spring-data-support')
```

Comment:
It doesn't look like you're using any classes from this project in your tests. Do you need this dependency?

Reply:
Not now in this PR, it is needed when adding audit metadata tests